CMakeLists.txt

```diff
@@ -111,4 +111,28 @@ IF(${PROJECT_NAME}_ENABLE_YouCompleteMe)
   INCLUDE(CodeCompletion)
 ENDIF()
 
+set(deprecated_packages Amesos AztecOO Epetra EpetraExt Ifpack Intrepid Isorropia ML Triutils)
```

Comment:
I probably should have asked earlier: what about PyTrilinos, Pliris, Zoltan?

Reply:
@ccober6 has the full list better than I.  I don't think Zoltan is deprecated.  PyTrilinos is a bit different because I don't know that it has been part of the deprecation announcements so far, but since PyTrilinos2 is now replacing it, we can add it in if that's what we'd like to do.  I don't have an opinion either way.

Reply:
Here are the packages that have a **required** dependency on Epetra: 
`'Amesos', 'AztecOO', 'EpetraExt', 'Ifpack', 'Isorropia', 'NewPackage', 'Pliris', 'ShyLU_DDCore', 'ThyraEpetraAdapters', 'ThyraEpetraExtAdapters'`

Here are the packages that have **optional** dependency on Epetra:
`'Amesos2', 'Anasazi', 'Belos', 'Galeri', 'ML', 'MueLu', 'NOX', 'PanzerDiscFE', 'PanzerDofMgr', 'Piro', 'PyTrilinos', 'ROL', 'ShyLU_DDFROSch', 'Teko', 'Teuchos', 'TpetraCore', 'Triutils', 'Xpetra', 'Zoltan2Core'`

There are **no** the packages whose _tests_ have **required** dependency on Epetra.

Here are the packages whose _tests_ have **optional** dependency on Epetra:
`'Ifpack2', 'Intrepid', 'Tempus', 'TrilinosCouplings', 'Xpetra', 'Zoltan2', 'Zoltan2Core'`

These packages we will be deprecating/archiving at the end of FY25:
`'Amesos', 'AztecOO', 'EpetraExt', 'Ifpack', 'Intrepid', 'Isorropia', 'ML', 'NewPackage', 'Pliris', 'ShyLU_DDCore', 'ThyraEpetraAdapters', 'ThyraEpetraExtAdapters', 'Triutils'`

Reply:
Okay, I'm updating the list now.  @cgcgcg what do you want to do for PyTrilinos?  I included it in the deprecations for now.

Reply:
Resolving and move conversation to #12746

Reply:
@sebrowne I'm cool with that